### DISCUSSION TITLE
Extração de whatsapp e outras coisas dados e uma GUI

### DISCUSSION THREAD

**Discussion by Lucasaschiefferdecker** [joz-k/ios_backup_extractor#10] [Ideas]
Eu criei uma GUI para está aplicação em C++ pretendo editar seu código em perl e adicionar está função! Podemos fazer isto juntos?

**Comment by joz-k** [comment]
I think this is great! However I would need your source code committed to some (Git) repository I can access. Qt/C++ is a good choice I think!

Also, if you clearly define what do I need to change in this command line tool (CLI) to be better controllable (usable) from the GUI app, I will gladly implement it.
